assets/collapsible-content.css

```diff
@@ -10,6 +10,10 @@
   }
 }
 
+.collapsible-content__media {
+  background-color: transparent;
+}
```

Comment:
> Oh actually I think it was needed so it doesn't show the shadow behind it 🤔 But we couldn't see it happen here because global settings is yet to be applied on this section.

We could give it a background colour that matches the background colour of the parent container so that it hides the shadow without compromising the transparency of the image.  If we add the `.gradient` class, it can even handle gradients that way.